migrator/migrations/m_172_to_m_173_network_flows_partition/migration.go

```diff
@@ -39,37 +38,44 @@ var (
 
 // MigrateToPartitions updates the btree network flow indexes to be hash
 func MigrateToPartitions(gormDB *gorm.DB, db *postgres.DB) error {
+	parentCtx := context.Background()
+
+	err := analyzeOldTable(parentCtx, db)
+	if err != nil {
+		log.WriteToStderrf("unable to analyze network_flows.  Will continue processing though it may be slow. %v", err)
+	}
+
 	// First get the distinct clusters in the network_flows table
-	clusters, err := getClusters(db)
+	clusters, err := getClusters(parentCtx, db)
```

Comment:
again curious if we shouldn't just read clusters here. Seems much faster

Reply:
I did.  But pulling in the store and all that nonsense seemed like overkill so I just did `getClustersStmt := "select distinct id from clusters;"` in `getClusters`